### PR TITLE
Language reference in markdown and LaTeX

### PR DESCRIPTION
**Pull request by zpcertik**
Typeset the language reference in Markdown, for nice display on Github, and create a Makefile that converts the Markdown to a PDF via LaTeX, so that future edits to the Markdown can be automatically reflected in the PDF. 

The Makefile uses `pandoc`, followed by `sed` to make some formatting substitutions. Right now, DeepSEA code blocks uses OCaml syntax highlighting, which looks good for 90% of code, but in a future update we could use custom highlighting (https://github.com/CertiKProject/deepsea/issues/26).

Perhaps this PR should be merged with the `deepsea` repo instead of this one? Please advise.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		${NAME}.tex
+	gsed -i '/.*hypertarget{introduction}.*/i \
+		\\\tableofcontents' \
+		${NAME}.tex
+	gsed -i 's/:exclamation:/ \
+		{\\fontencoding{U}\\fontfamily{futs}\\selectfont\\char 66\\relax}/' \
+		${NAME}.tex
+	pdflatex ${NAME}.tex
+	pdflatex ${NAME}.tex
+
```
Comment (jon-certik) [inline Makefile]: An html option?

**Comment by vilhelms** [comment]
Yeah, I think we should add the doc to the main deepsea repo (I guess on the `ethereum-proofless` branch for now) and then commit just the output to the deepsea-preview repo. I'm thinking the deepsea-preview repo is a temporary thing which will go away later in the year when we are ready to open-source the project.

**Comment by zpcertik** [comment]
OK, I'll migrate this PR to the `ethereum-proofless` on the main deepsea repo.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# brew install gnu-sed
+# brew install pandoc
+
+NAME="DeepSEA language reference"
+
+pdf: tex clean
+
+tex:
+	pandoc -s -o ${NAME}.tex ${NAME}.md
+	gsed -i 's/.*newenvironment{Shaded}.*/ \
```
[the remark this answers is not in the document]
Comment (zpcertik) [reply Makefile]: The fixed-width font for the code blocks sometimes overflows off the page, so this command modified the .tex file and declares the font `\small` within code blocks.

File: Makefile (hunk 1 above)
Comment (zpcertik) [reply Makefile]: Yes, I can do that now.

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (zpcertik) [reply Makefile]: The other two sed commands: add a table of contents; and replace the warning sings with something similar in LaTeX.

[the remark this answers is not in the document]
Comment (jon-certik) [reply Makefile]: Seems good now. For later use we can use pandoc's toc syntax and template customization.

**Comment by zpcertik** [comment]
Moved here: https://github.com/CertiKProject/deepsea/pull/28

**Comment by zpcertik** [comment]
OK, I've removed everything except the PDF from this PR. Should we still commit the new PDF, considering it's now linked to from our website?

**Comment by vilhelms** [comment]
Yeah, if we have a nicely formatted pdf, we should put it up, I think!

**Pull request merged**